app/javascript/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from "@reduxjs/toolkit";
+import greetingSlice from "./slice/greetingSlice";
```

Comment:
## [OPTIONAL]
- [ ] For a better redux stucture I would suggest you to put everything related to redux(slice, store) in a one dictory
![image](https://github.com/hasnatali1947/hello-rails-react2/assets/65068771/097dfed7-610c-49ca-a751-ca4f624ed8ab)